src/doom/p_saveg.h

```diff
@@ -62,5 +62,7 @@ void P_RestoreTargets (void);
 extern FILE *save_stream;
 extern boolean savegame_error;
 
+void  saveg_write8_in_stream  (FILE* stream, byte value , boolean* error);
+byte  saveg_read8_in_stream   (FILE* stream, boolean* error);
```

Comment:
These are not meant to stay here, they might get removed / moved into a namespace later on

Reply:
Why not putting them in a namespace directly then?

Reply:
Because the engine files are still in C, and there's no namespace in C